lib/homepage.rb

```diff
@@ -0,0 +1,35 @@
+class Homepage
+
+  attr_reader :currently_viewed_menu
+
+  def initialize
+    @restaurant_menus = []
+    @restaurants = []
+    @currently_viewed_menu = nil
+    @order = nil
+  end
+
+  def display_restaurants
```

Comment:
This feels like an addition to the project specs, which you should be careful about - trying to adhere as closely to the spec as possible is always a priority for projects & tech tests.